### DISCUSSION TITLE
Help setting up a custom domain

### DISCUSSION THREAD

**Discussion by Cristina-Assuncao** [community/community#64504] [New to GitHub] [answered]
Hi everyone

I encounter the following messages when trying to set up my custom page:
"Domain cristina-assuncao.com is not eligible for HTTPS at this time. " and "DNS check unsuccessful"
I am new to GitHub and although I read the GitHub Pages for Managing a custom domain I can't really understand what I am doing wrong!
Help is very welcome and needed!
Thank you

Cristina
<img width="805" alt="Screenshot 2023-08-21 at 15 58 08" src="https://github.com/community/community/assets/119290647/08b491bc-f593-4693-a3f2-6567d229fe49">

**Comment by arafajs** [comment, marked as answer]
The error message "Domain cristina-assuncao.com is not eligible for HTTPS at this time" means that your domain is not yet configured to use HTTPS. This can be due to a number of reasons, such as:

- Your domain registrar does not support HTTPS.
- You have not yet purchased an SSL certificate for your domain.
- You have not yet configured your DNS records to point to GitHub Pages.

To fix this error, you need to make sure that your domain is configured to use HTTPS. Here are the steps involved:

- Check with your domain registrar to see if they support HTTPS. If they do, you will need to purchase an SSL certificate from them.
- Once you have purchased an SSL certificate, you will need to configure your DNS records to point to GitHub Pages. You can find instructions on how to do this on the GitHub Pages documentation.

Once you have completed these steps, the error message should go away and your domain should be able to use HTTPS.

**Reply by Cristina-Assuncao** [reply]
Brilliant, thank you!

**Comment by cardy9049** [comment]
I'd appreciate any help you can give me on getting a new credit card from you soon and it is not limited to a $6,000.00 each week of getting out money at the teller in the bank and pay in cash..

**Comment by ismaiilkara** [comment]
Hello, I am taking the same error but my domain provider says there is no problem. Do you think that there would be other solutions by chance?
